apf/apfIPShape.cc

```diff
@@ -338,6 +362,12 @@ class LinearIPFit : public IPBase
         {
           fail("grads not implemented yet");
         }
+        void getVectorValues(Mesh*, MeshEntity*,
+            Vector3 const&,
+            NewArray<Vector3>&) const
+        {
+          fail("getVectorValues not defined for LinearIPFit shapes");
```

Comment:
Why not use `PCU_ALWAYS_ASSERT` here?

Reply:
This is to stay consistent with the surrounding code see for example like 163.

Reply:
ok